packages/core/lib/commands/debug/meta.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
       describe: "Force debugger to skip compilation (dangerous!)",
       type: "boolean",
       default: false
+    },
+    "url": {
+      describe: "Forces/set server URL",
```

Comment:
I think this description could use some work; also we would normally talk about a "provider" in this context rather than a "server".  Perhaps "Use specified URL for provider"?